### ISSUE TITLE
GoldenEye - mouse movement gains control at the wrong time

### ISSUE THREAD

**Issue by FitterSpace** [emoose/xenia#3]
From v1.9 onward, the player gains mouse control at the wrong time at the start of the level. Currently, you can only move the mouse when the level is completely done fading in. If you play with a controller, however, you can look around during the fade-in. This is also when the in-game timer starts.

**Comment by alkamaass** [comment]
yeah please please please fix this, gonna have to use rebinding with autohotkey with 1.8 instead of using the updated version, this is the one issue that has affected speedrunning

**Comment by emoose** [comment]
I've added a new 'ge_always_allow_inputs' config var in v1.13 which should allow mouse control to always be active, even during cutscenes etc, only temporary until I find a better way to detect the fade-in instead though.

**Comment by emoose** [comment]
Just released v1.13.2 which has a better method for detecting when camera-control is enabled, haven't tested it all that much yet though but seemed to work fine on the occasions I tried it.

If there's any issues let me know, if I don't hear anything then maybe 'ge_always_allow_inputs' can be removed in the next release after this.

**Comment by FitterSpace** [comment]
Thank you! This works great! I agree, you should be able to remove that option now that everything is working properly.

**Comment by FitterSpace** [comment]
Just found out that you can move the camera during the death animation, while the screen is turning red. The cinematic after that, where you see Bond dying from a 3rd-person perspective still locks the camera, as it should.
